### ISSUE TITLE
on kbox start, kb2 is trying to start container of another app?

### ISSUE THREAD

**Issue by labboy0276** [kalabox/kalabox#783]
Not sure if this is related to #782

However, when I went to start one app, it shoots out an error because it is trying to start the DB container of another app?  that can't be right:

```
john@kalamuna:~/kalabox/apps/cor777/code$ kbox start
info: DOCKER =>  Creating ad hoc container "kalabox/debian:stable". name=kalabox_temp-bf344444-64dc-45a0-89b4-328562809473, Cmd=[bash], Tty=true, Image=kalabox/debian:stable, Env=[APPDOMAIN=cor777.kbox, KBOX_CODEDIR=code, KALABOX=true, APPNAME=cor777, SSH_KEY=pantheon.kalabox.id_rsa, GITUSER=John Ouellet, GITEMAIL=john@kalamuna.com, PANTHEON_ACCOUNT=john@kalamuna.com], VolumesFrom=[kb_cor777_data]
info: DOCKER =>  Ad hoc container created. 4b4c26a873b8e307f8156b1d1fed710552c5a9a3e9de1c0cdc771bfd815dfd1c
info: DOCKER =>  Starting ad hoc container "4b4c26a873b8e307f8156b1d1fed710552c5a9a3e9de1c0cdc771bfd815dfd1c". Binds=[/Users/john/kalabox/apps/cor777:/src:rw]
info: DOCKER =>  Ad hoc container started. 4b4c26a873b8e307f8156b1d1fed710552c5a9a3e9de1c0cdc771bfd815dfd1c
info: DOCKER =>  Removing ad hoc container. 4b4c26a873b8e307f8156b1d1fed710552c5a9a3e9de1c0cdc771bfd815dfd1c
info: DOCKER =>  Starting container 28750929800d41749e12b530c0a1c156a7a822e967416f2c53b6dec374b46156. PublishAllPorts=true, Binds=[/Users/john/kalabox/apps/cor777:/src:rw]
info: DOCKER =>  Container started. 28750929800d41749e12b530c0a1c156a7a822e967416f2c53b6dec374b46156
info: DOCKER =>  Starting container 220323411bc44417cb2f41dd19bc701ec6ff36ae7d5a1e5fa7ece3c6f63dd236. PublishAllPorts=true, Binds=[/Users/john/kalabox/apps/cor777:/src:rw]
info: DOCKER =>  Container started. 220323411bc44417cb2f41dd19bc701ec6ff36ae7d5a1e5fa7ece3c6f63dd236
info: DOCKER =>  Starting container cec667fd5cd70610250fafed606fb6641722d9d16fac347abf5c8a421c23c840. PublishAllPorts=true, Binds=[/Users/john/kalabox/apps/cor777:/src:rw]
info: DOCKER =>  Container started. cec667fd5cd70610250fafed606fb6641722d9d16fac347abf5c8a421c23c840
info: DOCKER =>  Starting container 60faa4c241caca08ac5e8acf921b8741c99f46aba0fa6868b6d75d64915d6256. PublishAllPorts=true, Binds=[/Users/john/kalabox/apps/cor777:/src:rw]
info: DOCKER =>  Container started. 60faa4c241caca08ac5e8acf921b8741c99f46aba0fa6868b6d75d64915d6256
info: DOCKER =>  Starting container 4c4fe9cfc3b9517c835faf3080f03c5a0652c76a69cc4176301197477ce19b4f. PublishAllPorts=true, Binds=[/Users/john/kalabox/apps/cor777:/src:rw]
info: DOCKER =>  Container started. 4c4fe9cfc3b9517c835faf3080f03c5a0652c76a69cc4176301197477ce19b4f
Error dispatching event listeners [post-start]: Error inspecting container: kb_bakers-mission_db.: Error initializing docker.: Error getting docker config.: Error running command "ssh,-o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -p 2022 -i "/home/john/.ssh/boot2docker.kalabox.id_rsa" docker@localhost,sudo ifconfig eth1 10.13.37.42 netmask 255.255.255.0 broadcast 10.13.37.255 up".: code: 255, msg: 
VError: Error inspecting container: kb_bakers-mission_db.: Error initializing docker.: Error getting docker config.: Error running command "ssh,-o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -p 2022 -i "/home/john/.ssh/boot2docker.kalabox.id_rsa" docker@localhost,sudo ifconfig eth1 10.13.37.42 netmask 255.255.255.0 broadcast 10.13.37.255 up".: code: 255, msg: 
    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:290:13
From previous event:
    at Object.inspect (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:289:11)
From previous event:
    at /usr/local/bin/lib/engine.js.jx:422:29
From previous event:
    at Object.exports.inspect (/usr/local/bin/lib/engine.js.jx:421:4)
    at engine.list.map.concurrency (/usr/local/bin/lib/share.js.jx:33:19)
From previous event:
    at isAppRunning (/usr/local/bin/lib/share.js.jx:32:4)
    at /usr/local/bin/lib/share.js.jx:152:14
From previous event:
    at getDataContainers (/usr/local/bin/lib/share.js.jx:143:4)
    at getVolumes (/usr/local/bin/lib/share.js.jx:193:10)
    at Object.exports.restart (/usr/local/bin/lib/share.js.jx:416:17)
    at Object.fn (/usr/local/bin/plugins/kalabox-syncthing/events.js.jx:134:15)
    at /usr/local/bin/lib/core/events.js.jx:178:14
    at /usr/local/bin/lib/core/events.js.jx:177:24
From previous event:
    at /usr/local/bin/lib/core/events.js.jx:161:23
From previous event:
    at EventEngine._dispatch (/usr/local/bin/lib/core/events.js.jx:158:18)
    at /usr/local/bin/lib/core/events.js.jx:137:19
From previous event:
    at EventEngine.emit (/usr/local/bin/lib/core/events.js.jx:129:21)
From previous event:
    at Object.exports.emit (/usr/local/bin/lib/core/events.js.jx:261:4)
    at /usr/local/bin/lib/app.js.jx:759:24
From previous event:
    at Object.exports.start (/usr/local/bin/lib/app.js.jx:758:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-core/tasks/start.js.jx:24:18)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:876:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:170:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:233:12)
    at process._tickCallback (node.js:812:13)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:232:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:241:1)
    at Module._compile (module.js:518:26)
    at readX (module.js:1284:5)
    at Object.Module._extensions..jx (module.js:1711:10)
    at Module.load (module.js:345:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:377:17)
    at require (module.js:395:17)
    at Object.<anonymous> (_jx_source:17:3)
    at Module._compile (module.js:518:26)
    at Module.load (module.js:353:12)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:733:12)
    at startup (node.js:371:18)
    at node.js:1464:3
info: BOOT2DOCKER =>  Starting up. max=3
info: BOOT2DOCKER =>  Initializing boot2docker [1].
john@kalamuna:~/kalabox/apps/cor777/code$
```

**Comment by leevh** [comment]
Seemingly same error has been crippling my kbox2 too.  It's happening [post-start] and [post-stop]

```
info: DOCKER =>  Starting container 83a4c730f30a4522ecb92397f837e340fcf1e95c9cae267d69dc83e3d4daa404. PublishAllPorts=true, Binds=[/Users/leevanh/kalabox/web-191:/src:rw]
info: DOCKER =>  Container started. 83a4c730f30a4522ecb92397f837e340fcf1e95c9cae267d69dc83e3d4daa404
Error dispatching event listeners [post-start]: Error inspecting container: kb_web-350_edge.: Error initializing docker.: Error getting docker config.: Error running command "ssh,-o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -p 2022 -i "/home/leevanh/.ssh/boot2docker.kalabox.id_rsa" docker@localhost,sudo ifconfig eth1 10.13.37.42 netmask 255.255.255.0 broadcast 10.13.37.255 up".: code: 255, msg: 
VError: Error inspecting container: kb_web-350_edge.: Error initializing docker.: Error getting docker config.: Error running command "ssh,-o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -p 2022 -i "/home/leevanh/.ssh/boot2docker.kalabox.id_rsa" docker@localhost,sudo ifconfig eth1 10.13.37.42 netmask 255.255.255.0 broadcast 10.13.37.255 up".: code: 255, msg: 
    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:290:13
From previous event:
    at Object.inspect (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:289:11)
From previous event:
    at /usr/local/bin/lib/engine.js.jx:422:29
From previous event:
    at Object.exports.inspect (/usr/local/bin/lib/engine.js.jx:421:4)
    at engine.list.map.concurrency (/usr/local/bin/lib/share.js.jx:33:19)
From previous event:
    at isAppRunning (/usr/local/bin/lib/share.js.jx:32:4)
    at /usr/local/bin/lib/share.js.jx:152:14
From previous event:
    at getDataContainers (/usr/local/bin/lib/share.js.jx:143:4)
    at getVolumes (/usr/local/bin/lib/share.js.jx:193:10)
    at Object.exports.restart (/usr/local/bin/lib/share.js.jx:416:17)
    at Object.fn (/usr/local/bin/plugins/kalabox-syncthing/events.js.jx:134:15)
    at /usr/local/bin/lib/core/events.js.jx:178:14
    at /usr/local/bin/lib/core/events.js.jx:177:24
From previous event:
    at /usr/local/bin/lib/core/events.js.jx:161:23
From previous event:
    at EventEngine._dispatch (/usr/local/bin/lib/core/events.js.jx:158:18)
    at /usr/local/bin/lib/core/events.js.jx:137:19
From previous event:
    at EventEngine.emit (/usr/local/bin/lib/core/events.js.jx:129:21)
From previous event:
    at Object.exports.emit (/usr/local/bin/lib/core/events.js.jx:261:4)
    at /usr/local/bin/lib/app.js.jx:759:24
From previous event:
    at exports.start (/usr/local/bin/lib/app.js.jx:758:4)
    at /usr/local/bin/lib/app.js.jx:862:12
From previous event:
    at Object.exports.restart (/usr/local/bin/lib/app.js.jx:861:4)
    at Object.task.func (/usr/local/bin/plugins/kalabox-core/tasks/restart.js.jx:15:18)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:876:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:170:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:233:12)
    at process._tickCallback (node.js:812:13)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:232:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:241:1)
    at Module._compile (module.js:518:26)
    at readX (module.js:1284:5)
    at Object.Module._extensions..jx (module.js:1711:10)
    at Module.load (module.js:345:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:377:17)
    at require (module.js:395:17)
    at Object.<anonymous> (_jx_source:17:3)
    at Module._compile (module.js:518:26)
    at Module.load (module.js:353:12)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:733:12)
    at startup (node.js:371:18)
    at node.js:1464:3
```
and
```
Error dispatching event listeners [post-stop]: Error inspecting container: kb_web-350_redis.: Error querying docker for list of containers.: Error initializing docker.: Error getting docker config.: Error running command "ssh,-o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -p 2022 -i "/home/leevanh/.ssh/boot2docker.kalabox.id_rsa" docker@localhost,sudo ifconfig eth1 10.13.37.42 netmask 255.255.255.0 broadcast 10.13.37.255 up".: code: 255, msg: 
VError: Error inspecting container: kb_web-350_redis.: Error querying docker for list of containers.: Error initializing docker.: Error getting docker config.: Error running command "ssh,-o IdentitiesOnly=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=quiet -p 2022 -i "/home/leevanh/.ssh/boot2docker.kalabox.id_rsa" docker@localhost,sudo ifconfig eth1 10.13.37.42 netmask 255.255.255.0 broadcast 10.13.37.255 up".: code: 255, msg: 
    at /usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:290:13
From previous event:
    at Object.inspect (/usr/local/bin/node_modules/kalabox-engine-docker/docker.js.jx:289:11)
From previous event:
    at /usr/local/bin/lib/engine.js.jx:422:29
From previous event:
    at Object.exports.inspect (/usr/local/bin/lib/engine.js.jx:421:4)
    at engine.list.map.concurrency (/usr/local/bin/lib/share.js.jx:33:19)
From previous event:
    at isAppRunning (/usr/local/bin/lib/share.js.jx:32:4)
    at /usr/local/bin/lib/share.js.jx:152:14
From previous event:
    at getDataContainers (/usr/local/bin/lib/share.js.jx:143:4)
    at getVolumes (/usr/local/bin/lib/share.js.jx:193:10)
    at Object.exports.restart (/usr/local/bin/lib/share.js.jx:416:17)
    at Object.fn (/usr/local/bin/plugins/kalabox-syncthing/events.js.jx:130:15)
    at /usr/local/bin/lib/core/events.js.jx:178:14
    at /usr/local/bin/lib/core/events.js.jx:177:24
From previous event:
    at /usr/local/bin/lib/core/events.js.jx:161:23
From previous event:
    at EventEngine._dispatch (/usr/local/bin/lib/core/events.js.jx:158:18)
    at /usr/local/bin/lib/core/events.js.jx:137:19
From previous event:
    at EventEngine.emit (/usr/local/bin/lib/core/events.js.jx:129:21)
From previous event:
    at Object.exports.emit (/usr/local/bin/lib/core/events.js.jx:261:4)
    at /usr/local/bin/lib/app.js.jx:833:24
From previous event:
    at exports.stop (/usr/local/bin/lib/app.js.jx:832:4)
    at Object.exports.restart (/usr/local/bin/lib/app.js.jx:859:10)
    at Object.task.func (/usr/local/bin/plugins/kalabox-core/tasks/restart.js.jx:15:18)
    at Object.exports.run (/usr/local/bin/lib/core/tasks.js.jx:876:15)
    at processTask (/usr/local/bin/bin/kbox.js.jx:170:16)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:233:12)
    at process._tickCallback (node.js:812:13)
From previous event:
    at main (/usr/local/bin/bin/kbox.js.jx:232:4)
    at Object.<anonymous> (/usr/local/bin/bin/kbox.js.jx:241:1)
    at Module._compile (module.js:518:26)
    at readX (module.js:1284:5)
    at Object.Module._extensions..jx (module.js:1711:10)
    at Module.load (module.js:345:36)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:377:17)
    at require (module.js:395:17)
    at Object.<anonymous> (_jx_source:17:3)
    at Module._compile (module.js:518:26)
    at Module.load (module.js:353:12)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:733:12)
    at startup (node.js:371:18)
    at node.js:1464:3
```

**Comment by pirog** [comment]
I think this should be resolved in v0.11
